s3-lambda-comprehend-cdk-python/DEVELOPMENT.md

```diff
@@ -0,0 +1,32 @@
+
+### Project Structure within s3-lambda-comprehend-cdk-batch-detect-sentiment:
+```
+s3-lambda-comprehend-cdk-python
+- app.py
+- cdk.json
+- src/
+   - lambda_function.py
+   - architecture.png
+   - user_input_text.txt
+- requirements.txt
+- source.bat
```

Comment:
where is this file or can that line be deleted?

Reply:
Line removed.